frontend/src/Components/Elements/Table/Table.js

```diff
@@ -33,6 +34,19 @@ function Table({
       paginated ? usePagination : ''
     );
 
+  const displayMissingPhrase = (ds) => {
+    const phrase = {
+      STOPS: 'stops',
+      SEARCHES: 'searches',
```

Comment:
I see `undefined` on the searches by percentage data table:

![image](https://user-images.githubusercontent.com/129679/223566545-924450be-4a0c-4ac0-8031-b848cbf45d3e.png)

Reply:
Good catch, missing that one. Updated